test/gui/tst_addAccount/test.feature

```diff
@@ -80,3 +80,17 @@ Feature: adding accounts
         And the user syncs the "Personal" space
         Then the folder "simple-folder" should exist on the file system
 
+
+    Scenario: Check for incremented number in bracket while adding new account and folder
+        Given folder "ownCloud" has been created in local system
+        And the user has started the client
+        And the user has entered the following account information:
+            | server | %local_server% |
+        When the user adds the following user credentials:
```

Comment:
maybe we can refactor existing step: `the user adds the following wrong user credentials:`